app/views/recipes/index.html.erb

```diff
@@ -1,14 +1,22 @@
-<p style="color: green"><%= notice %></p>
-
-<h1>Recipes</h1>
-
-<div id="recipes">
-  <% @recipes.each do |recipe| %>
-    <%= render recipe %>
-    <p>
-      <%= link_to "Show this recipe", recipe %>
-    </p>
+<div class="container bg-dark text-light rounded-top px-4 pb-2 mb-4">
+  <div class="container d-flex gap-5">
+    <div class="ms-auto m-1">
+      <%= button_to "Add new recipe", new_recipe_path, method: :get, class: "btn btn-warning mt-10 mb-10 text-end m-3 me-auto order-last" %>
+    </div>
+  </div>
+  
+  <% @recipes.each do |recipe|%>
+  <div class='shadow-lg p-3 mb-5 bg-body-tertiary rounded bg-danger d-flex'>
+    <div class='card-body'>
+      <h3 class="card-title text-decoration-none m-3 text-dark"><%= recipe.name %></h3> 
+      <div class="d-flex gap-3"> 
+        <h3><%= button_to "Remove", recipe, method: :delete,  class: 'btn btn-warning m-2 text-end', data: { turbo: false }  %></h3>
+        <h3><%= link_to "Show details", recipe_path(recipe.id), class: "btn btn-warning mt-10 m-2 text-decoration-none" %></h3>
+      </div>
+    </div>
+    <div class="card-body m-2 w-50">
+      <p class="text-dark"  style="overflow-wrap: break-word;"><%= recipe.description %><p>
+    </div>
+  </div>
```

Comment:
- [ ] Great job so far @ridabensalem and @prius2055 There are just a few changes I will suggest you implement and you'll be on your way to the next milestone :+1: 
- [ ] From the image below and since you are a team of less than 3 members, you were supposed to implement these 5 pages below. But currently, I only have access to 3 which are the `food index page`, the `recipe index page`, and the `recipe show page` :+1: 
- [ ] You're missing the `public recipes page` and the `general shopping list page` so kindly implement those as well. :+1: 
- [ ] However, I noticed that you did create those page pages, but they weren't in your `navbar` as a result there was no way for a user to know that those pages exist. However, I checked your routes and navigated to the `public_recipe index page` and the `general_shopping_list page` but they both had errors I have attached them below. So kindly fix that as well and ensure that they work as expected and match the given wireframes :wink:
- [ ] Lastly, kindly add them to your `Navbar` to allow users to open all of the pages. :+1: 
- The given wireframe :+1: 
![Screenshot from 2023-08-11 15-44-58](https://github.com/ridabensalem/recipe-app/assets/102699958/dda2a3e5-b187-4463-84c4-d804ea5520b6)

- Error Message on the `public_recipes_page` 
![image](https://github.com/ridabensalem/recipe-app/assets/102699958/2736775e-2981-42a5-aa0c-1e8c082689d4)

- Error Message on the `general_shopping_list` 
![image](https://github.com/ridabensalem/recipe-app/assets/102699958/5cbc08da-53ed-4e12-8f98-3d2cd46e6067)

Reply:
Thank you. This has been fixed

Reply:
Fixed.